files/en-us/web/css/calc-constant/index.md

```diff
@@ -14,55 +14,45 @@ browser-compat: css.types.calc-constant
 
 {{CSSRef}}
 
-The **`<calc-constant>`** [CSS](/en-US/docs/Web/CSS) [data type](/en-US/docs/Web/CSS/CSS_Types) represents well-defined constants such as `e` and `π`. Rather than require authors to manually type out several digits of these constants, a few of them are provided directly by CSS.
+The **`<calc-constant>`** [CSS](/en-US/docs/Web/CSS) [data type](/en-US/docs/Web/CSS/CSS_Types) represents common constants such as `e` and `pi`.
+These constants are supported for convenience in CSS so that authors do not have to manually write or calculate them.
+Mathematical constants can only be used inside [CSS math functions](/en-US/docs/Web/CSS/CSS_Functions#math_functions) for calculations.
+If used outside of a calculation, they're treated like any other keyword.
+For example:
+
+- `animation-name: pi;` refers to an animation named "pi", not the `pi` numeric constant.
+- `line-height: e;` is invalid, but `line-height: calc(e);` is valid.
+
+Using `infinity` or `-infinity` is intended to be more convenient for authors to create large fake rather than calculating or generating them through other means.
+Although `infinity` and `NaN` are not technically numbers, they are evaluated as {{cssxref("number")}} values, so an expression like `calc(infinity * 1px)` can be used to create an infinite {{cssxref("length")}} value, for example.
```

Comment:
I think the bit about "they are evaluated as {{cssxref("number")}} values, so an expression like `calc(infinity * 1px)` can be used to create an infinite {{cssxref("length")}} value, for example" is true about all these constants, so it is misleading to hitch it to "Although `infinity` and `NaN` are not technically numbers".

I don't really think "Although `infinity` and `NaN` are not technically numbers" is helpful. What does "not technically numbers" mean?

```suggestion
In math functions, `<calc-constant>` values are evaluated as {{cssxref("number")}} values, so an expression like `calc(infinity * 1px)` can be used to create an infinite {{cssxref("length")}} value, for example.
```